### PR TITLE
Exclude the .yarn folder from policheck

### PR DESCRIPTION
**Pull request by RyanToth3**

### CODE REVIEW

**Comment by RyanToth3** [comment]
Validating suppression works properly on a build before completing

**Comment by RyanToth3** [comment]
Validated that the exclusion is applied properly

File: azure-pipelines/PoliCheckExclusions.xml [hunk 1]
```diff
@@ -6,5 +6,5 @@
     <!--Each of these file types will be completely skipped for the entire scan -->
     <!-- <Exclusion Type="FileType">.ABC|.XYZ</Exclusion>-->
     <!--The specified file names will be skipped during the scan regardless which folder they are in -->
-    <!-- <Exclusion Type="FileName">ABC.TXT|XYZ.CS</Exclusion>-->
+    <Exclusion Type="FileName">LICENSE.TXT</Exclusion>
```
Comment (AArnott) [inline azure-pipelines/PoliCheckExclusions.xml:9]: Where is there a `LICENSE.TXT` file? Not in this repo, unless it's under the yarn cache somewhere.

Comment (RyanToth3) [reply azure-pipelines/PoliCheckExclusions.xml:9]: src/servicebroker-npm/LICENSE.txt

**Pull request merged**